airflow/cli/commands/role_command.py

```diff
@@ -41,3 +39,259 @@ def roles_create(args):
     appbuilder = cached_app().appbuilder  # pylint: disable=no-member
     for role_name in args.role:
         appbuilder.sm.add_role(role_name)
+
+
+mapping = {
+    # ========================= Airflow Views
+    ("Airflow", "can_blocked"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_clear"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_delete")],
+    ("Airflow", "can_code"): [("Dag", "can_read"), ("DagCode", "can_read")],
+    ("Airflow", "can_dag_details"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_dag_stats"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_dagrun_clear"): [
+        ("Dag", "can_read"),
+        ("Task", "can_read"),
+        ("TaskInstance", "can_delete"),
+    ],
+    ("Airflow", "can_dagrun_failed"): [("Dag", "can_read"), ("DagRun", "can_edit")],
+    ("Airflow", "can_dagrun_success"): [("Dag", "can_read"), ("DagRun", "can_edit")],
```

Comment:
These two also edit TaskInstances I think - so do we assume that can_edit on a DagRun implies can_edit on those task instances?